zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/IdentitySetupInitializeTest.java

```diff
@@ -147,7 +176,14 @@ public void shouldNotCreateRoleIfAlreadyExists() {
     engine.role().newRole(roleId).create().getKey();
 
     // when
-    engine.identitySetup().initialize().withRole(role).withUser(user).initialize();
+    engine
+        .identitySetup()
+        .initialize()
+        .withRole(role)
+        .withUser(user)
+        .withRoleMember(
```

Comment:
❓ I can see that we've added the role member parts to the tests but I can't see tests for the other memberships (I may be missing something here though). As this behaviour is vital for operations, I would add tests for it